pyvelox/pyvelox.h

```diff
@@ -504,6 +527,56 @@ static void addVectorBindings(
             std::move(baseVector),
             PyVeloxContext::getSingletonInstance().pool());
       });
+
+  m.def(
+      "row_vector",
+      [](std::vector<std::string>& names,
+         std::vector<VectorPtr>& values,
+         const std::optional<py::list>& nullability_list) {
+        std::vector<std::shared_ptr<const Type>> childTypes;
+        childTypes.resize(values.size());
+        for (int i = 0; i < values.size(); i++) {
+          childTypes[i] = values[i]->type();
+        }
+        auto rowType = ROW(std::move(names), std::move(childTypes));
+        const size_t vectorSize = values.size();
+        BufferPtr nullability_buffer = nullptr;
+        if (nullability_list.has_value()) {
+          auto nullability_values = nullability_list.value();
+          nullability_buffer = AlignedBuffer::allocate<bool>(
+              nullability_values.size(),
```

Comment:
nullability size can then be different than vector size, they should all be one size, irrespective of number of elements in nullability_list...